lib/utils/fast-boot.js

```diff
@@ -0,0 +1,22 @@
+const { getParent } = require('../utils/common');
+
+function isEnvironmentBrowser(node) {
+  let callee = node.test.callee;
+
+  if (callee) {
+    let obj = callee.object;
+    let prop = callee.property;
+
+    return obj.name === 'environment' && prop.name === 'isBrowser';
```

Comment:
In the long run (once FastBoot 1.0 releases), the support for `environment.isBrowser()` will be dropped. It would be better to check for [isFastboot](https://github.com/ember-fastboot/ember-cli-fastboot/blob/master/app/services/fastboot.js#L92)